dt/uni04delta/nncp/kustomization.yaml

```diff
@@ -0,0 +1,523 @@
+    targets:
+      - select:
+          kind: NodeNetworkConfigurationPolicy
+        fieldPaths:
+          - spec.desiredState.interfaces.[name=storagenfs].vlan.base-iface
+  - source:
+      kind: ConfigMap
+      name: network-values
+      fieldPath: data.storagenfs.vlan
+    targets:
+      - select:
+          kind: NodeNetworkConfigurationPolicy
+        fieldPaths:
+          - spec.desiredState.interfaces.[name=storagenfs].vlan.id
+  - source:
+      kind: ConfigMap
+      name: network-values
+      fieldPath: data.storagenfs.mtu
+    targets:
+      - select:
+          kind: NodeNetworkConfigurationPolicy
+        fieldPaths:
+          - spec.desiredState.interfaces.[name=storagenfs].mtu
+  # ctlplane type is ethernet (not vlan)
+  - source:
+      kind: ConfigMap
+      name: network-values
+      fieldPath: data.ctlplane.iface
+    targets:
+      - select:
+          kind: NodeNetworkConfigurationPolicy
+        fieldPaths:
+          - spec.desiredState.interfaces.[type=ethernet].name
+          - spec.desiredState.interfaces.[type=linux-bridge].bridge.port.0.name
+  - source:
+      kind: ConfigMap
+      name: network-values
+      fieldPath: data.ctlplane.mtu
+    targets:
+      - select:
+          kind: NodeNetworkConfigurationPolicy
+        fieldPaths:
+          - spec.desiredState.interfaces.[type=ethernet].mtu
+          - spec.desiredState.interfaces.[type=linux-bridge].mtu
+
+  # Static Node IPs: node-0
+  - source:
+      kind: ConfigMap
+      name: network-values
+      fieldPath: data.node_0.internalapi_ip
+    targets:
+      - select:
+          kind: NodeNetworkConfigurationPolicy
+          name: node-0
+        fieldPaths:
+          - spec.desiredState.interfaces.[name=internalapi].ipv4.address.0.ip
+  - source:
+      kind: ConfigMap
+      name: network-values
+      fieldPath: data.node_0.tenant_ip
+    targets:
+      - select:
+          kind: NodeNetworkConfigurationPolicy
+          name: node-0
+        fieldPaths:
+          - spec.desiredState.interfaces.[name=tenant].ipv4.address.0.ip
+  - source:
+      kind: ConfigMap
+      name: network-values
+      fieldPath: data.node_0.ctlplane_ip
+    targets:
+      - select:
+          kind: NodeNetworkConfigurationPolicy
+          name: node-0
+        fieldPaths:
+          - spec.desiredState.interfaces.[type=linux-bridge].ipv4.address.0.ip
+  - source:
+      kind: ConfigMap
+      name: network-values
+      fieldPath: data.node_0.storage_ip
+    targets:
+      - select:
+          kind: NodeNetworkConfigurationPolicy
+          name: node-0
+        fieldPaths:
+          - spec.desiredState.interfaces.[name=storage].ipv4.address.0.ip
+  - source:
+      kind: ConfigMap
+      name: network-values
+      fieldPath: data.node_0.storagenfs_ip
+    targets:
+      - select:
+          kind: NodeNetworkConfigurationPolicy
+          name: node-0
+        fieldPaths:
+          - spec.desiredState.interfaces.[name=storagenfs].ipv4.address.0.ip
```

Comment:
```
  stderr: 'error: accumulating components: accumulateDirectory:
  "recursed accumulation of path
  ''/home/zuul/src/github.com/openstack-k8s-operators/architecture/dt/uni04delta/nncp'':
  fieldPath `data.node_0.storagenfs_ip` is missing for replacement source
  ConfigMap.[noVer].[noGrp]/network-values.[noNs]"'
```

Reply:
The newly proposed network name does not have "storage" in it since kustomize only supports substring matching and not full regex. Thus, "storagenfs" and "storage" match when they should not and the kustomization does not work.